wagtail/admin/static_src/wagtailadmin/js/core.js

```diff
@@ -83,6 +83,57 @@ function enableDirtyFormCheck(formSelector, options) {
 }
 
 $(function() {
+    // Load icon sprite and cache it in local storage.
+    function loadIconSprite() {
+        var spriteURL = wagtailConfig.ADMIN_URLS.SPRITE;
+        var revisionKey = 'spriteRevision';
+        var dataKey = 'spriteData';
+
+        // Feature check
+        if (!document.createElementNS || !document.createElementNS('http://www.w3.org/2000/svg', 'svg').createSVGRect) {
+            return true;
+        }
```

Comment:
It looks like we don’t need this, since all our target browsers support SVG (https://caniuse.com/#search=createSVGRect).